styles.css

```diff
@@ -454,6 +454,207 @@ input[type=name] {
   align-items: center;
 }
 
+#work-section {
+  font-family: sans-serif;
+  font-weight: 500;
+  margin-top: 70px;
+}
+
+#box-1 {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  margin-top: 40px;
+}
+
+#head {
+  border-bottom: 1px solid #646369;
+}
+
+#head h2 {
+  width: 50%;
+  font-size: 40px;
+  line-height: 54px;
+  color: #7d73fc;
+  margin-bottom: 30px;
+}

```

Comment:
- [ ] Please use the same color used in the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346), and kindly remove the fixed width, to match the design as well. 😁